sdk/src/main/java/com/kin/ecosystem/balance/presenter/IBalancePresenter.java

```diff
@@ -2,22 +2,9 @@
 
 import com.kin.ecosystem.balance.view.IBalanceView;
 import com.kin.ecosystem.base.IBasePresenter;
-import com.kin.ecosystem.main.ScreenId;
 
-public interface IBalancePresenter extends IBasePresenter<IBalanceView> {
-
-	interface BalanceClickListener {
-
-		void onClick();
-	}
 
-	void onStart();
-
-	void onStop();
-
-	void balanceClicked();
+public interface IBalancePresenter extends IBasePresenter<IBalanceView> {
```

Comment:
1. Why a marker(empty) interface?
2. where did all those method went?
For example, why did you move addObservers() to onAttach() and removeObservers() to onDetach?
What was the problem in the old way?
3. why you didn't made this interface also in Kotlin?

Reply:
1. to be aligned with all views
2. Before the change the presenter was initialized on the EcoActivity, now it's on the BalanceView itself.
3. not sure will do :)